### ISSUE TITLE
Blank page when importing package

### ISSUE THREAD

**Issue by ouhzzzman** [nusu/avvvatars#4]
Hi there, this is just the standard App.js file that is initiliazed to which I imported the Avvvatars package. When I save the code and refresh the page, it turns blank (so no rendering). Note that I'm just importing the package and not using it either.

```
import logo from './logo.svg';
import './App.css';
import Avvvatars from 'avvvatars-react';

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <p>
          Edit <code>src/App.js</code> and save to reload.
        </p>
        <a
          className="App-link"
          href="https://reactjs.org"
          target="_blank"
          rel="noopener noreferrer"
        >
          Learn React
        </a>
        <p>
          
        </p>
      </header>
    </div>
  );
}

export default App;
```

I checked whether the package has been properly installed and it's appearing in package.json too.

**Comment by nusu** [comment]
Hey, 
I tried to reproduce this but I don't get the same results, if it's caused by the package you should at least see an error on console.

I'm gonna need a little bit more information on that topic to inspect what's wrong, such as what boilerplate did you use, your package.json content, and if it's possible a codesandbox link to quickly review it.

**Comment by ouhzzzman** [comment]
You're right, this is the error I'm getting from the console 
```
Uncaught TypeError: goober.styled is not a function
    at Object../node_modules/avvvatars-react/dist/avvvatars-react.cjs.development.js (Shape.tsx:18:1)
    at Object.options.factory (react refresh:6:1)
    at __webpack_require__ (bootstrap:24:1)
    at fn (hot module replacement:61:1)
    at Object../node_modules/avvvatars-react/dist/index.js (index.js:7:1)
    at Object.options.factory (react refresh:6:1)
    at __webpack_require__ (bootstrap:24:1)
    at fn (hot module replacement:61:1)
    at Module../src/App.js (logo.svg:34:1)
    at Module.options.factory (react refresh:6:1)
```

**Comment by cgen01** [comment]
I'm having the same issue on my end

**Comment by nusu** [comment]
Still can't reproduce the issue, which node version and react version have you used? and is there any chance that you can send me a demo on github or codesanbox

**Comment by ouhzzzman** [comment]
I'm using version 16.14.0 for node and 17.0.2 for react. 
Quick demo: https://www.loom.com/share/7110faeb849348bda23eced44f63eba3

**Comment by nusu** [comment]
Thanks for letting me know, I've managed to reproduce it and fixed it with #7, you can install latest version and let me know if it works.
